src/main/java/com/fasterxml/jackson/databind/deser/BasicDeserializerFactory.java

```diff
@@ -468,6 +468,10 @@ private boolean _addExplicitDelegatingCreator(DeserializationContext ctxt,
         int ix = -1;
         final int argCount = candidate.paramCount();
         SettableBeanProperty[] properties = new SettableBeanProperty[argCount];
+        if (argCount == 0) {
+            creators.addPropertyCreator(candidate.creator(), true, properties);
```

Comment:
It's not entirely clear if we should use a property creator with no properties, or `creators.setDefaultCreator(candidate.creator());`. The former seems more explicit, but the latter more closely matches the previous implementation.

Reply:
I agree with your choice; let's go with that (possible to change, too, if need arises).